database/buildSchema/30_review_status_history.sql

```diff
@@ -0,0 +1,37 @@
+-- review section join
+
+
+CREATE TYPE public.review_status as ENUM ('Draft', 'Submitted', 'Changes Required', 'Review Pending');
+
+CREATE TABLE public.review_status_history (
+	id serial primary key,
+	review_id integer references public.review(id),
+	"status" public.review_status,
+	"timestamp" timestamp with time zone,
+	is_current boolean DEFAULT true
+);
+
+-- FUNCTION to set `is_current` to false on all other review_status_histories of current review
```

Comment:
Not important for now, but we should look at generalising this function at some point (since its the same as the application ones) to take in "table name" as additional parameter.

Reply:
I think it's fine to have this method duplicated. It seems a good trade off for readability, for now anyways.